lib/server.js

```diff
@@ -83,10 +83,11 @@ function CoAPServer(options, listener) {
   // 32 MB / 1280 = 26214
   // The max lifetime is roughly 200s per packet.
   // Which gave us 131 packets/second guarantee
+  this._lruMaxAge = (parameters.exchangeLifetime * 1000)
   this._lru = LRU({
       max: options.cacheSize || (32768 * 1024)
-    , length: function(n) { return n.length }
-    , maxAge: parameters.exchangeLifetime
+    , length: function(n) { return n.buffer.byteLength }
+    , maxAge: this._lruMaxAge
```

Comment:
OK. I'll switch to an Interval called once every maxAge that runs a prune() across the whole lru.